packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md

```diff
@@ -0,0 +1,353 @@
+
+### Assign users and groups in the Org2Org app
+
+In each spoke org, assign the users and groups to the Org2Org app integration by using the [Okta Apps API application operations](/docs/reference/api/apps/#application-user-operations):
+
+* [`POST /api/v1/apps/${yourOrg2OrgAppId}/users` (assign user to an application for SSO and provisioning)](/docs/reference/api/apps/#assign-user-to-application-for-sso-and-provisioning)
+* [`POST /api/v1/apps/${yourOrg2OrgAppId}/groups/${groupId}` (assign group to an application)](/docs/reference/api/apps/#assign-group-to-application)
+
+Alternatively, you can assign users and groups for provisioning using the Okta Admin Console. See [Assign an app integration to a user](https://help.okta.com/okta_help.htm?type=oie&id=ext-lcm-assign-app-user) and [Assign an app integration to a group](https://help.okta.com/okta_help.htm?type=oie&id=ext-lcm-assign-app-groups).
+
+After you've assigned your users or groups in the spoke org, validate that the same users or groups appear in your hub org.
+
+## Key rotation
+
+An advantage to using the OAuth 2.0 connection is that you can [rotate keys](/docs/concepts/key-rotation) to adhere to cryptographic best practices. You can rotate keys for a specific OAuth 2.0 connection by following these API steps:
+
+1. [Generate a new key for the Org2Org app](#generate-a-new-key-for-the-org2org-app)
+2. [Register the new key with the corresponding service app](#register-the-new-org2org-app-key-with-the-corresponding-service-app)
+3. [Update the current credentials for the Org2Org app](#update-the-current-credentials-for-the-org2org-app)
+
+> **Note**: If you want to minimize downtime during key rotation, you can update the service app (step two) with both the old and new keys, since `jwks.keys` is an array that can handle different `kid` identifiers. You can remove the old key after you've verified that provisioning works with the new key.
+
+### Generate a new key for the Org2Org app
+
+From your spoke org, make a request to [generate a new application key credential](/docs/reference/api/apps/#generate-new-application-key-credential) as an Okta admin user.
+
+##### Request example
+
+```bash
+curl -X POST \
+  -H 'Accept: application/json' \
+  -H "Authorization: SSWS ${spokeApiToken}" \
+  -H 'Content-Type: application/json' \
+  "https://${yourSpokeOrgDomain}/api/v1/apps/${yourOrg2OrgAppId}/credentials/keys/generate?validityYears=${validYears}"
+```
+
+##### Response example
+
+```json
+{
+    "kty": "RSA",
+    "created": "2022-01-20T19:50:14.000Z",
+    "lastUpdated": "2022-01-20T19:50:14.000Z",
+    "expiresAt": "2024-01-20T19:50:13.000Z",
+    "kid": "sf-jWwRKMUU5588aokhj-xu_mGucHLxIh_-fYLAofB8",
+    "use": "sig",
+    "x5c": [
```

Comment:
nit - the formatting of the x5c section should be indented a bit.

Reply:
Do you want the whole "x5c" line indented? or just the line below?
Btw, I copied this JSON from Postman response and I just shortened the long strings.

Reply:
It's fine - or maybe move the bracket up to the same line as the key? But, it's not incorrect - just that closing bracket threw me. I'm good now. I've had coffee.